Content.Shared/Lathe/SharedLatheSystem.cs

```diff
@@ -59,6 +61,11 @@ public bool CanProduce(EntityUid uid, LatheRecipePrototype recipe, int amount =
         return true;
     }
 
+    private void OnEmagged(EntityUid uid, LatheComponent component, ref GotEmaggedEvent args)
+    {
+        args.Handled = true;
+    }
```

Comment:
this should be a subscription on the EmagLatheRecipesComponent, as emagged a regular lathe with no extra recipes shouldn't do anything.